content/020_prerequisites/workspace.md

```diff
@@ -10,6 +10,10 @@ not the root account user. Please ensure you are logged in as an IAM user, not t
 account user.
 {{% /notice %}}
 
+{{% notice warning %}}
+If you are at an AWS Hosted Event Cloud 9 Environment is already built for you. Just open the existing IDE in the Cloud9 console.
```

Comment:
Think you're missing a "the" in front of "Cloud 9 Environment", could you add and re-submit please? Apart from that, looks good!

Reply:
Added it. Will submit the pull request again.